<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="j-shangguan" />

<meta name="date" content="2021-01-15" />

<title>Literature Review</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">WinterRotationNotebook</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/j-shangguan/WinterRotationNotebook">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Literature Review</h1>
<h4 class="author">j-shangguan</h4>
<h4 class="date">2021-01-15</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2021-01-31
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 2 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>WinterRotationNotebook/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomjshangguanWinterRotationNotebooktreef1ece3b684db46f03cbbb44993da6d2cec714842targetblankf1ece3ba"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/j-shangguan/WinterRotationNotebook/tree/f1ece3b684db46f03cbbb44993da6d2cec714842" target="_blank">f1ece3b</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomjshangguanWinterRotationNotebooktreef1ece3b684db46f03cbbb44993da6d2cec714842targetblankf1ece3ba" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/j-shangguan/WinterRotationNotebook/tree/f1ece3b684db46f03cbbb44993da6d2cec714842" target="_blank">f1ece3b</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/.DS_Store
    Ignored:    analysis/images/.DS_Store
    Ignored:    data/.DS_Store

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/literature.Rmd</code>) and HTML (<code>docs/literature.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/j-shangguan/WinterRotationNotebook/blob/f1ece3b684db46f03cbbb44993da6d2cec714842/analysis/literature.Rmd" target="_blank">f1ece3b</a>
</td>
<td>
j-shangguan
</td>
<td>
2021-01-31
</td>
<td>
new edits
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/j-shangguan/WinterRotationNotebook/e72301986c4f7d45d12d23bc3d0242914cc06170/docs/literature.html" target="_blank">e723019</a>
</td>
<td>
j-shangguan
</td>
<td>
2021-01-31
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/j-shangguan/WinterRotationNotebook/blob/9c39a8cb90890099dd9d96d7d0162cc2353a0218/analysis/literature.Rmd" target="_blank">9c39a8c</a>
</td>
<td>
j-shangguan
</td>
<td>
2021-01-31
</td>
<td>
added new pages, revised old ones
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/j-shangguan/WinterRotationNotebook/5775459a71bfeb79a4191c3a0ef3024ca7526946/docs/literature.html" target="_blank">5775459</a>
</td>
<td>
j-shangguan
</td>
<td>
2021-01-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/j-shangguan/WinterRotationNotebook/f3c54ac2b475f39e99ebc0ae53171420c09b9e63/docs/literature.html" target="_blank">f3c54ac</a>
</td>
<td>
j-shangguan
</td>
<td>
2021-01-15
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/j-shangguan/WinterRotationNotebook/blob/b7f94cd518a3c5c8c4ecafafd8abbb2c94f13b4b/analysis/literature.Rmd" target="_blank">b7f94cd</a>
</td>
<td>
j-shangguan
</td>
<td>
2021-01-15
</td>
<td>
added new background notes and lit review
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="condensates" class="section level2">
<h2><strong>Condensates</strong></h2>
<div id="shorter-j.-the-mammalian-disaggregase-machinery-hsp110-synergizes-with-hsp70-and-hsp40-to-catalyze-protein-disaggregation-and-reactivation-in-a-cell-free-system.-plos-one.-2011610e26319.-httpsdoi.org10.1371journal.pone.0026319" class="section level3">
<h3>Shorter J. The mammalian disaggregase machinery: Hsp110 synergizes with Hsp70 and Hsp40 to catalyze protein disaggregation and reactivation in a cell-free system. PLoS One. 2011;6(10):e26319. <a href="https://doi.org/10.1371/journal.pone.0026319" class="uri">https://doi.org/10.1371/journal.pone.0026319</a></h3>
<ul>
<li>In yeast: Hsp104
<ul>
<li>Hsp104 is a AAA+ ATPase that breaks apart large assemblies</li>
<li>Hsp104 works with Hsp70 and Hsp40 for protein disaggregation
<ul>
<li>Recovers enzymatically active proteins from aggregates</li>
<li>Circumvents costs of needing to degrade protein and replace it (transcription, translation)</li>
</ul></li>
<li>In yeast:
<ul>
<li>Hsp110= Sse1; Sse1 regulates ATP hydrolysis of Hsp70</li>
<li>Hsp70 = Ssa1</li>
<li>Hsp40 =Sis1, Ydj1</li>
</ul></li>
</ul></li>
<li>No Hsp104 in animals/Metazoa
<ul>
<li>Hsp110 works with chaperones Hsp70 and Hsp40 to drive disaggregation in mammals</li>
<li>In mammals:
<ul>
<li>Hsp110= Apg-2</li>
<li>Hsp70 = Hsc70, Hsp70</li>
<li>Hsp40 =Hdj1</li>
</ul></li>
</ul></li>
<li>Cooperation between these components is not species limited
<ul>
<li>Hsp104 can work with mammalian chaperones</li>
</ul></li>
</ul>
</div>
<div id="cinar-h.-winter-r.-the-effects-of-cosolutes-and-crowding-on-the-kinetics-of-protein-condensate-formation-based-on-liquidliquid-phase-separation-a-pressure-jump-relaxation-study.-sci-rep-10-17245-2020.-httpsdoi.org10.1038s41598-020-74271-x" class="section level3">
<h3>Cinar, H., Winter, R. The effects of cosolutes and crowding on the kinetics of protein condensate formation based on liquid–liquid phase separation: a pressure-jump relaxation study. Sci Rep 10, 17245 (2020). <a href="https://doi.org/10.1038/s41598-020-74271-x" class="uri">https://doi.org/10.1038/s41598-020-74271-x</a></h3>
<ul>
<li>Metastable between binodal (coexistence) and spinodal curve
<ul>
<li>Below spinodal curve is two-phase region</li>
</ul></li>
<li>Phase separation processes are different in binodal vs spinodal region
<ul>
<li>Metastable region
<ul>
<li>Nucleation and growth</li>
<li>Tiny droplets of form, grow via solute diffusion, and combine</li>
<li>Rate of nucleation is determined by energy barrier of making boundary surface of new phase
<ul>
<li>If thermal fluctuations allow droplet to grow to critical size where SA costs &lt;volume energy returns, then nucleation occurs</li>
<li>After droplet formed, solute from local area go into droplet</li>
</ul></li>
</ul></li>
<li>Spinodal decomposition
<ul>
<li>Spontaneous formation of domains of higher and lower concentrations</li>
</ul></li>
</ul></li>
</ul>
</div>
<div id="walters-rw-muhlrad-d-garcia-j-parker-r.-differential-effects-of-ydj1-and-sis1-on-hsp70-mediated-clearance-of-stress-granules-in-saccharomyces-cerevisiae.-rna.-20152191660-1671.-httpsdoi10.1261rna.053116.115" class="section level3">
<h3>Walters RW, Muhlrad D, Garcia J, Parker R. Differential effects of Ydj1 and Sis1 on Hsp70-mediated clearance of stress granules in Saccharomyces cerevisiae. RNA. 2015;21(9):1660-1671. <a href="https://doi:10.1261/rna.053116.115" class="uri">https://doi:10.1261/rna.053116.115</a></h3>
<ul>
<li>System: yeast</li>
<li>Background
<ul>
<li>mRNP granule (messenger ribonucleoprotein)
<ul>
<li>Types
<ul>
<li>P-bodies
<ul>
<li>Translationally repressed mRNAs, translational repressor, and mRNA degradation machinery</li>
<li>mRNA either is: stored, degraded, returned to translation</li>
</ul></li>
<li>Stress granules
<ul>
<li>Nontranslating mRNAs, mRNA binding proteins, translation initiation machinery</li>
<li>mRNA is stalled during translation and then recovered</li>
</ul></li>
</ul></li>
<li>Granule assembly potentially promoted by intrinsically disordered domains</li>
</ul></li>
<li>Results
<ul>
<li>Hsp70 and Hsp40 proteins colocalize with stress granules
<ul>
<li>Ded1-mCherry: labeling stress granule resident protein</li>
</ul></li>
<li>Defects in Hsp70 or Hsp40 inhibit stress granule disassembly during stress recovery
<ul>
<li>Ydj1 defects inhibited translation recovery –&gt; vacuole stress granule accumulation</li>
<li>Sis1 defects did not affect translation –&gt; cytoplasmic stress granule accumulation</li>
</ul></li>
</ul></li>
</ul></li>
</ul>
</div>
<div id="paper-cite" class="section level3">
<h3>2015 paper (cite)</h3>
<ul>
<li>Main takeaway: heat shock induced protein aggregates are reversible and have biological activity</li>
<li>Aggregates: pellatable proteins that are soluble pre-heat shock</li>
<li>Imaging in Fig.2
<ul>
<li>C-terminal fluorescently tagged yeast at their native chromosomal loci</li>
<li>Yeast diploid strains: mRuby2, Pab1-Clover</li>
<li>Imaged at 30 deg. C</li>
<li>8 min. heat shock at 46 deg. C</li>
<li>Results
<ul>
<li>Control: After heat shock, Non-aggregating glycolytic enzyme Pgk1-mRuby2 is cytosolic and diffuse</li>
<li>Pab1-mRuby2 and Pab1-Clover form cytosolic foci</li>
<li>Components of the multisynthetase complex colocalize with Pab1
<ul>
<li>Mes1 and Gus1 colocalize with Pab1</li>
<li>Arc1 colocalizes with Gus1</li>
<li>Ola1 colocalizes with Pab1</li>
<li>Four heat-shock granule components: Mes1, Gus1, Arc1, Ola1, where
<ul>
<li>AME complex: heterotrimer of Arc1, Mes1, Gus1 that interact via N-terminal domains
<ul>
<li>Arc1: aminoacylation cofactor</li>
<li>Mes1: Methionyl-tRNA synthetase</li>
<li>Gus1: glutamyl-tRNA synthetase</li>
</ul></li>
</ul></li>
<li>Fig5: AME is stoichiometric complex</li>
</ul></li>
<li>Fig6. Disaggregation of heat-shock granules after 1 hr recovery</li>
<li>Imaging done with spinning-disk confocal fluorescence microscopy</li>
</ul></li>
</ul></li>
</ul>
</div>
<div id="paper-cite-1" class="section level3">
<h3>2017 paper (cite)</h3>
<ul>
<li>Demixing = phase separation</li>
<li>Main takeaways
<ul>
<li>Temperature and pH affect Pab1 phase separation</li>
<li>RNA inhibits phase separation</li>
<li>P domain promotes demixing but is not required</li>
<li>Phase separation contributes to stress tolerance and adaptive thermal sensitivity</li>
</ul></li>
<li>Quinary structures: no fixed stoichiometry
<ul>
<li>Often multivalent interactions, e.g. between LCRs
<ul>
<li>Low-complexity regions (LCRs): contain higher freq of a certain set of AA</li>
</ul></li>
</ul></li>
<li>Pab1 structure in budding yeast
<ul>
<li>Short N-terminal leader sequence
<ul>
<li>Leader sequence: between promoter and coding region, regulate gene expression</li>
<li>More about leader sequences: Mølhøj, M., Degan, F. Leader sequences are not signal peptides. Nat Biotechnol 22, 1502 (2004). <a href="https://doi.org/10.1038/nbt1204-1502" class="uri">https://doi.org/10.1038/nbt1204-1502</a></li>
</ul></li>
<li>4 RNA-recognition motifs (RRMs)</li>
<li>P domain: proline-rich LCR; intrinsically disordered</li>
<li>C-terminal peptide binding domain</li>
</ul></li>
<li>Results
<ul>
<li>Fig1. Confocal fluorescence microscopy
<ul>
<li>Diploid strains: Pab1-mRuby2, Pab1-Clover</li>
<li>8 minute incubation at temp.</li>
<li>Controls: no Pab1 in stress granules at 42 deg. C (or 30)</li>
<li>10 min heat shock,46 deg. C: Pab1 localizes to stress granules</li>
</ul></li>
<li>Fig.2: looking at Pab1 demixing in vitro
<ul>
<li>Purified recombinant yeast Pab1 from E coli</li>
<li>Pab1 is monomeric in shifts up to 33 deg. C - Shift to 36 deg. and above accelerates Pab1 demixing</li>
<li>2F: measuring radial growth length</li>
</ul></li>
<li>Fig. 3: “Pab1 quinary assemblies form by phase separation and gel formation”
<ul>
<li>Demixing is affected by ionic strength and pH
<ul>
<li>High salt concentrations –&gt; inhibited demixing at 46 deg</li>
<li>pH&gt;7 –&gt; inhibited demixing at 46 deg</li>
</ul></li>
<li>Making Pab1 quinary assemblies
<ul>
<li>Pab1-mRub2 in unlabeled Pab1 solution, 1:20<br />
</li>
<li>Demixing, then imaged at room temperature within 1-2 min
<ul>
<li>Heat-induced demixing: branched clusters of spherical droplets</li>
<li>pH-induced demixing: spherical droplets</li>
<li>What if both? –&gt; Does changing pH, then changing temp cause branched clustering?</li>
</ul></li>
</ul></li>
<li>Forming two-layer droplet
<ul>
<li>Clover-Pab1 in unlabeled Pab1, 1:20</li>
<li>Trigger phase separation, then wait 5 min</li>
<li>Added Pab1-mRub2 in unlabeled Pab1 solution, 1:20</li>
<li>Result: after 24 hours, layers did not mix</li>
</ul></li>
</ul></li>
</ul></li>
</ul>
<p>Include notes from Wallace et al. (2015), Kroschwald et al. (2015)</p>
</div>
</div>
<div id="pab1-and-condensates" class="section level2">
<h2><strong>Pab1 and Condensates</strong></h2>
<div id="yoo-h-triandafillou-c-drummond-da.-cellular-sensing-by-phase-separation-using-the-process-not-just-the-products.-j-biol-chem.-2019294187151-7159.-httpsdoi.org10.1074jbc.tm118.001191" class="section level3">
<h3>Yoo H, Triandafillou C, Drummond DA. Cellular sensing by phase separation: Using the process, not just the products. J Biol Chem. 2019;294(18):7151-7159. <a href="https://doi.org/10.1074/jbc.TM118.001191" class="uri">https://doi.org/10.1074/jbc.TM118.001191</a></h3>
<ul>
<li>Phase separation, then assembly process of phase-separated subassemblies
<ul>
<li>Distinction between formation process and final product</li>
</ul></li>
<li>Liquid-liquid phase separation: single liquid phase transitions to 2 or more distinct liquid phases</li>
<li>Phase separation allows for fast system-wide changes in repsonse to stimuli, so that matter is rearranged quickly and potentially without an energy cost</li>
<li>Two important properties of phase separation in relation to sensing
<ul>
<li>Phase separations are cooperative</li>
<li>Phase changes allow for cellular matter to be rearranged with less time and energy cost compared to de novo synthesis of mRNA or protein and degradation</li>
</ul></li>
<li>Environmental temperature changes have large effects on system-wide behavior
<ul>
<li>Most organisms live within a narrow thermal range</li>
<li>When temp rises, heat-shock proteins are produced</li>
<li>Eukaryotes form stress granules in the case of high heat shock
<ul>
<li>Stress granules = RNA + protein clusters in cytosol</li>
</ul></li>
</ul></li>
<li>Pab1 in yeast: a highly conserved RNA-binding protein found in stress granules
<ul>
<li>In response to thermal stress, Pab1 undergoes phase separation and forms a hydrogel</li>
<li>Hydrophobic interactions in proline-rich domain (P domain) modulate phase boundary</li>
<li>RNA-binding domains mediate phase separation</li>
<li>Pab’s phase separation is a very thermosensitive process
<ul>
<li>Pab1 can sense a 3% change in aboslute temperature</li>
</ul></li>
<li>pH also affects Pab’s phase separation
<ul>
<li>pH drop occurs with heat shock</li>
</ul></li>
<li>Pab1 most likely use thermal and intracellular pH information to sense size of thermal stress</li>
<li>When Pab1’s phase separation is inhibited, growth is inhibited –&gt; phase separation is adapative</li>
<li>Potential function of Pab1’s phase separation
<ul>
<li>Translation regulation:
<ul>
<li>during stress-triggered phase separation, Pab1 releases heat-shock mRNAs</li>
<li>repress heat-shock mRNAs by binding to their A-rich 5’ UTRs before/after stress</li>
</ul></li>
</ul></li>
<li>Pab1 needs molecular chaperones to disperse demixed molecules after environment returns to intial state</li>
</ul></li>
<li>Nucleation and growth vs spinodal decomposition processes
<ul>
<li>Phase separation can happen in 2 ways, and each have offer a different biological control mechanism</li>
<li>Nucleation and growth
<ul>
<li>First, energetically unfavorable nucleation step occurs –&gt; nuclei formed</li>
<li>Second, spontaneous growth of the dense phase of nuclei</li>
<li>Each step is regulated separately, and controlling where nuclei are formed affects location of phase-separated structures</li>
</ul></li>
<li>Spinodal decomposition
<ul>
<li>First, spontaneous nucleation –&gt; nuclei appear throughout solution</li>
<li>Second, system spontaneously separates</li>
<li>Synchronized system-wide switch</li>
</ul></li>
<li>Pab1 is in the nucleation and growth regime (Riback 2017)
<ul>
<li>Pab1 prefers to form assemblies from exisitng assembliies</li>
</ul></li>
</ul></li>
</ul>
</div>
</div>
<div id="superresolution-imaging-of-condensates" class="section level2">
<h2><strong>Superresolution Imaging of Condensates</strong></h2>
<div id="jain-s-wheeler-jr-walters-rw-agrawal-a-barsic-a-parker-r.-atpase-modulated-stress-granules-contain-a-diverse-proteome-and-substructure.-cell.-20161643487-498.-httpsdoi.org10.1016j.cell.2015.12.038" class="section level3">
<h3>Jain S, Wheeler JR, Walters RW, Agrawal A, Barsic A, Parker R. ATPase-Modulated Stress Granules Contain a Diverse Proteome and Substructure. Cell. 2016;164(3):487-498. <a href="https://doi.org/10.1016/j.cell.2015.12.038" class="uri">https://doi.org/10.1016/j.cell.2015.12.038</a></h3>
<ul>
<li>Anti-GFP-AF647 for STORM imaging of GFP-G3BP in stress granules</li>
<li>Induced stress granules with sodium azide and then lysed to get yeast cell lysate</li>
<li>Mammalian stress granules show regions of high G3BP, PABP!, or polyA RNA
<ul>
<li>~200 nm in diameter in stress granules</li>
<li>This “core” is surrounded by lower concentrations of other components, “shell”
<ul>
<li>“Shell” appears to be dynamic, while “core” is more stable</li>
</ul></li>
</ul></li>
</ul>
</div>
<div id="wang-jt-smith-j-chen-bc-et-al.-regulation-of-rna-granule-dynamics-by-phosphorylation-of-serine-rich-intrinsically-disordered-proteins-in-c.-elegans.-elife.-20143e04591.-published-2014-dec-23.-httpsdoi.org10.7554elife.04591" class="section level3">
<h3>Wang JT, Smith J, Chen BC, et al. Regulation of RNA granule dynamics by phosphorylation of serine-rich, intrinsically disordered proteins in C. elegans. Elife. 2014;3:e04591. Published 2014 Dec 23. <a href="https://doi.org/10.7554/eLife.04591" class="uri">https://doi.org/10.7554/eLife.04591</a></h3>
<ul>
<li>Lattice sheet microsopy on GFP-MEG3 and mCherry-PGL3 in P granules in C elegans</li>
<li>MEG3 surrounds P granule</li>
<li>Images not super clear…</li>
</ul>
</div>
<div id="pab1-labeled-with-alexa-fluor-488-or-alexa-fluor-647-carboxylic-acid-succinimidyl-ester-dyes-for-confocal-microscopy-as-done-here-httpsdoi.org10.11012020.08.07.234443" class="section level3">
<h3>Pab1 labeled with Alexa Fluor 488 or Alexa Fluor 647 carboxylic acid succinimidyl ester dyes for confocal microscopy as done here: <a href="https://doi.org/10.1101/2020.08.07.234443" class="uri">https://doi.org/10.1101/2020.08.07.234443</a></h3>
</div>
<div id="wheeler-jr-matheny-t-jain-s-abrisch-r-parker-r.-distinct-stages-in-stress-granule-assembly-and-disassembly.-elife.-20165e18413.-published-2016-sep-7.-httpsdoi.org10.7554elife.18413" class="section level3">
<h3>Wheeler JR, Matheny T, Jain S, Abrisch R, Parker R. Distinct stages in stress granule assembly and disassembly. Elife. 2016;5:e18413. Published 2016 Sep 7. <a href="https://doi.org/10.7554/eLife.18413" class="uri">https://doi.org/10.7554/eLife.18413</a></h3>
<ul>
<li>Stress granule assembly disassembly in mammalian cells primarily, also yeast</li>
<li>Focus is on interactions among IDR driving phase separation</li>
<li>Another SIM paper of GFP-G3BP1</li>
<li>Stress induced via NaAsO2 (sodium arsenate) in mammalian system</li>
<li>Low temperature inhibits mammalian stress granule assembly</li>
<li>Although supposedly low temperature promotes LLPS, authors propose that granule assembly is from mRNPs oligomerizing into stable core complexes
<ul>
<li>Core structure contains then a local high concentration of IDRs, triggereing local phase separation</li>
</ul></li>
</ul>
</div>
<div id="wei-jie-zeng-chuxin-lu-yuanyuan-shi-chunyan-wu-xinxin-chen-chunmei-li-jie-yao-initiation-of-stress-granule-assembly-by-rapid-clustering-of-igf2bp-proteins-upon-osmotic-shock-biochimica-et-biophysica-acta-bba---molecular-cell-research-2020-httpsdoi.org10.1016j.bbamcr.2020.118795." class="section level3">
<h3>Wei-jie Zeng, Chuxin Lu, Yuanyuan Shi, Chunyan Wu, Xinxin Chen, Chunmei Li, Jie Yao, Initiation of stress granule assembly by rapid clustering of IGF2BP proteins upon osmotic shock, Biochimica et Biophysica Acta (BBA) - Molecular Cell Research, 2020, <a href="https://doi.org/10.1016/j.bbamcr.2020.118795" class="uri">https://doi.org/10.1016/j.bbamcr.2020.118795</a>.</h3>
<ul>
<li>Dual-color STORM of recruitment of IGF2BP1 and polyA mRNA to stress granule during osmotic stress</li>
<li>Fixed U2OS cells: mEos2-IGF2BP1 (first imaged), polyA mRNA detected via AF647-oligodT probe</li>
</ul>
</div>
<div id="fei-j-jadaliha-m-harmon-ts-et-al.-quantitative-analysis-of-multilayer-organization-of-proteins-and-rna-in-nuclear-speckles-at-super-resolution.-j-cell-sci.-2017130244180-4192.-httpsdoi.org10.1242jcs.206854" class="section level3">
<h3>Fei J, Jadaliha M, Harmon TS, et al. Quantitative analysis of multilayer organization of proteins and RNA in nuclear speckles at super resolution. J Cell Sci. 2017;130(24):4180-4192. <a href="https://doi.org/10.1242/jcs.206854" class="uri">https://doi.org/10.1242/jcs.206854</a></h3>
<ul>
<li>Example of what super res can tell us
<ul>
<li>See distinct multilayer organization of proteins and RNAs in condensates
<ul>
<li>Non-random organization of subset of nuclear speckle components
<ul>
<li>Functionally sub-compartmentalized: transcription and splicing in speckle periphery, RNA maturation in inner part of domain</li>
</ul></li>
<li>Quantify distribution of protein and RNA components in speckle to model order of operations that lead to speckle functions</li>
</ul></li>
</ul></li>
<li>Findings
<ul>
<li>3D SIM of FISH and IF staining showed that protein SC35 localizes in core of nuclear speckles while essential spliceosome component U2 snRNA is peripherally distributed</li>
<li>Knock down and observe changes in gene expression and speckle organization</li>
<li>RNA accumulation in speckle preferentially localizes to specific layers within nuclear speckles</li>
<li>Can tune condensate components to alter morphology and organization of speckles</li>
</ul></li>
</ul>
</div>
<div id="niewidok-b-igaev-m-pereira-da-graca-a-strassner-a-lenzen-c-richter-cp-piehler-j-kurre-r-brandt-r.-2018.-single-molecule-imaging-reveals-dynamic-biphasic-partition-of-rna-binding-proteins-in-stress-granules.-j-cell-biol-217-13031318.-httpsdoi.org10.1083jcb.201709007" class="section level3">
<h3>Niewidok B, Igaev M, Pereira da Graca A, Strassner A, Lenzen C, Richter CP, Piehler J, Kurre R, Brandt R. 2018. Single-molecule imaging reveals dynamic biphasic partition of RNA-binding proteins in stress granules. J Cell Biol 217: 1303–1318. <a href="https://doi.org/10.1083/jcb.201709007" class="uri">https://doi.org/10.1083/jcb.201709007</a></h3>
<ul>
<li>System: stressed nueronal cells
<ul>
<li>Fluorescence decay after photoactivation (FDAP) measurements and single moluecle localization microscopy
<ul>
<li>Observe protein exchange between stress granules and protein diffusion/interactions within stress granules</li>
</ul></li>
<li>Imaging tells whether distribution of components within stress granules is responsible for sorting dynamics of stress granules</li>
</ul></li>
<li>G3BP1 is a constitutive stress granule protein; it is a RNA-binding protein Ras GTPase-activating protein Sh3-domain-binding protein 1
<ul>
<li>Involved in regulating assembly and function of stress granules</li>
</ul></li>
<li>IMP1 is another RNA binding protein that is found in stress granules</li>
<li>G3BP1 and IMP1 were used as representative proteins of ribonucleoprotein granules
<ul>
<li>Photoactivatable GFP-tagged fusion proteins</li>
</ul></li>
<li>Hotspots of G3BP1 and IMP1 in stress granules</li>
</ul>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
